src/mobile/src/libs/crypto.js

```diff
@@ -24,7 +26,14 @@ export const getRandomBytes = async (quantity) => {
 
 export const generatePasswordHash = async (password, salt) => {
     const salt64 = await encodeBase64(salt);
-    return getHashFn()(password, salt64, DEFAULT_ARGON2_PARAMS).then(
+    if (isAndroid) {
+        return getHashFn()(values(password), salt64, DEFAULT_ARGON2_PARAMS).then(
+            (result) => Uint8Array.from(result),
+            (error) => console.log(error), // eslint-disable-line no-console
```

Comment:
I don't think this is necessary here. If there's an error it would automatically propagated across the promise chain.

Reply:
But there isn't always a catch block when this function is called.